### ISSUE TITLE
Confusing bounding box key name

### ISSUE THREAD

**Issue by bilzard** [yueruchen/Pedestrian-Synthesis-GAN#31]
bounding box's key names 'h' and 'w' make us associate 'height' and 'width', but they are **NOT**.

according source code, they means bounding box's bottom right (x, y) coordinate.
https://github.com/yueruchen/Pedestrian-Synthesis-GAN/blob/master/models/pix2pix_model.py#L93-L94

(This repository seems to be not active, but I left this note for the future reference)

**Comment by yifanjiang19** [comment]
yes, you are right. I will leave this issue open to notice other users. Thanks!
